### PR TITLE
H1 changes for SEO

### PR DESCRIPTION
**Pull request by lhcole613**
FREELANCERS: Please do NOT merge this PR as per @pebaum

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

File: microsoft-365/admin/admin-overview/what-subscription-do-i-have.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ description: "Learn to verify which subscriptions your organization has by going
 ms.date: 01/21/2021
 ---
 
-# What subscription do I have?
+# What Microsoft 365 subscription do I have?
```
Comment (SKjerland) [inline microsoft-365/admin/admin-overview/what-subscription-do-i-have.md]: @lhcole613 - This should be "Which Microsoft 365 subscription do I have?"

File: microsoft-365/admin/add-users/admin-roles-page.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ search.appverid:
 description: "Admin roles map to business functions and provide permissions to do specific tasks in the admin center. For example, the Service admin opens support tickets with Microsoft."
 ---
 
-# Get started with the roles page
+# Get started with the admin roles page
```
Comment (SKjerland) [inline microsoft-365/admin/add-users/admin-roles-page.md]: @lhcole613 - I know we like to include the product name in H1 headings for SEO, so shouldn't this be "Get started with the Microsoft 365 admin roles page"?

File: microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ ms.assetid: 1dbcf12f-a9de-4d1d-b0b3-a227f8a736d8
 description: "Give a user the right to access another user's mailbox, which allows the user to read and send emails from the other user's mailbox."
 ---
 
-# Give mailbox permissions to another user - Admin Help
+# Give mailbox permissions to another user - Admin help
```
Comment (SKjerland) [inline microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md]: @lhcole613 - Don't we need to include the product name in the heading? And is there a reason why we add "- Admin help" to this H1? We don't typically add that to H1 headings for other admin topics, do we?

File: microsoft-365/admin/add-users/resend-user-password.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ ms.assetid: 9b838071-94aa-4b8e-8d84-d17ece1ee951
 description: "Learn how to reset password and send a email to a new Microsoft 365 user."
 ---
 
-# Resend a user's password - Admin Help
+# Resend a user's password - Admin help
```
Comment (SKjerland) [inline microsoft-365/admin/add-users/resend-user-password.md]: @lhcole613 - Same comment as above about product name and wondering why "- Admin help" is added.

**Review by SKjerland — CHANGES_REQUESTED** [review]
@lhcole613 - I made some comments inline. Thanks!

File: microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md (hunk 3 above)
Comment (kwekuako) [reply microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md]: @SKjerland, we added these to the end of Docs topics that endusers were landing on and then getting confused. Not sure what we're doing with those topics now.

File: microsoft-365/admin/add-users/resend-user-password.md (hunk 4 above)
Comment (kwekuako) [reply microsoft-365/admin/add-users/resend-user-password.md]: @SKjerland, same as above.

File: microsoft-365/admin/add-users/admin-roles-page.md (hunk 2 above)
Comment (lhcole613) [reply microsoft-365/admin/add-users/admin-roles-page.md]: Yes. Not sure that happened there :-).

File: microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md (hunk 3 above)
Comment (lhcole613) [reply microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md]: Sounds like a bigger conversation, but I would like to add the product name in the H1, but placement here is interesting. Does this work? Give mailbox permissions to another Microsoft 365 user - Admin help

File: microsoft-365/admin/add-users/resend-user-password.md (hunk 4 above)
Comment (lhcole613) [reply microsoft-365/admin/add-users/resend-user-password.md]: Suggest addition of: Resend a Microsoft 365 user's password

File: microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md (hunk 3 above)
Comment (SKjerland) [reply microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md]: @lhcole613 - That looks good to me, Leslie! Thanks!

File: microsoft-365/admin/add-users/resend-user-password.md (hunk 4 above)
Comment (SKjerland) [reply microsoft-365/admin/add-users/resend-user-password.md]: @lhcole613 - Looks good!

**Review by pebaum — APPROVED** [review]

**Pull request merged**